src/witan/cic/episodes.clj

```diff
@@ -357,6 +335,77 @@
   (x/by-key ::id
             (x/reduce ssda903-split-headers-and-episodes)))
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; remove placement m1, m2, m3 as they are discontinued.
+;;
+;; If the report_date of e2 for a M* episode is the same as the ceased
+;; of e1, then the ceased of e1 should be the ceased of e2 and e2
+;; should be marked for removal
+;;
+;; Given an episode
+;; When the placment code is the discontinued M1, M2, or M3
+;; Then mark the episode for removal
+;;
+;; Assumption: CYP are not missing at the beginning of a period but
+;; only at the end or middle of a period of care
```

Comment:
So if I understand correctly M* episdoes can be "fixed" if there is additional data, but otherwise they are removed?

Reply:
They aren't removed at all atm. I'm making the assumption the M* will only occur in the middle of a period, as the type has been discontinued since 2015.